### PR TITLE
Issue #16: create CSS samples

### PR DESCRIPTION
**Pull request by arp82**
Created the basic sample for CSS and CSS in JS integration with Next.js

### CODE REVIEW

**Comment by brauliodiez** [comment]
Feedback:

- It would be nice as well if you can provide a nice style to the table, just grab it from some sample, e.g. this one looks simple:

https://jsfiddle.net/zinoui/dB93J/

https://codepen.io/Harrix/pen/zLLQYy?page=1&

File: 07-styles/readme.md [hunk 1]
```diff
@@ -1,135 +1,168 @@
...
 
-- Now we will make an update to pass to the user-info url the name of the user instead of the id.
+- In order to use external CSS files in our application, we actually need to add a new dependency to our project. Concretely, we should run the code below to add the ```next-css``` package to our app:
+```bash
+npm install --save @zeit/next-css
+```
 
-_./pages/components/user-collection/row.tsx_
+- This new dependency provides us with a ```withCSS``` function that we can invoke in our ```next.config.js``` file to extend our current configuration with CSS-handling capabilities. Thus, should refactor our ```next.config.js``` as follows:
+```javascript
```
Comment (brauliodiez) [inline 07-styles/readme.md]: missing _fullpath/filename_

Comment (arp82) [reply 07-styles/readme.md]: Sure, I will be reviewing this ASAP

**Comment by brauliodiez** [comment]
Good stuff let's merge this, and ask for a merge into main repo master

**Pull request merged**